lxd/storage/drivers/driver_ceph.go

```diff
@@ -122,6 +115,17 @@ func (d *ceph) Create() error {
 		}
 	}
 
+	return nil
+}
+
+// Create is called during pool creation and is effectively using an empty driver struct.
+// WARNING: The Create() function cannot rely on any of the struct attributes being set.
+func (d *ceph) Create() error {
+	revert := revert.New()
```

Comment:
I think we should be calling FillConfig from inside Create, as it already can modify the config.